hadoop-hdds/docs/content/feature/Quota.md

```diff
@@ -63,6 +63,31 @@ e. Linked bucket will consume namespace quota of volume.
 
 f. If the cluster is upgraded from old version less than 1.1.0, use of quota on older volumes and buckets(We can confirm by looking at the info for the volume or bucket, and if the quota value is -2 then volume or bucket is old) is not recommended. Since the old key is not counted to the bucket's namespace quota, the quota setting is inaccurate at this point.
 
+g. For files and directory moving to trash, trash will consume extra namespace for below cases:
```

Comment:
I would suggest adding “For FSO bucket” at the beginning of the sentence.

Reply:
@ChenSammi changes updated.